### DISCUSSION TITLE
Image File Thumbnail Preview

### DISCUSSION THREAD

**Discussion by hosamn** [d2phap/ImageGlass#735] [General] [answered]
Is it at all possible to add icon preview of supported file types to windows explorer...?!

**Comment by hosamn** [comment]
thumbnail preview is what I mean.

**Comment by d2phap** [comment, marked as answer]
By default, Windows supports some basic image formats, but you can installed some Shell extensions to Windows:
For RAW: https://www.microsoft.com/store/productId/9NCTDW2W1BH8

**Comment by hosamn** [comment]
and I found this to preview SVG :
https://github.com/tibold/svg-explorer-extension

But we need something more solid and diverse,
and preferably integrated with our most loved image viewer :)

**Comment by d2phap** [comment]
There is a very old request for this at #196 
I will take a look some times
